src/pages/workspace/tags/WorkspaceEditTagsPage.tsx

```diff
@@ -52,10 +52,17 @@ function WorkspaceEditTagsPage({route}: WorkspaceEditTagsPageProps) {
         [translate, policyTags, route.params.orderWeight],
     );
 
-    const goBackToTagsSettings = useCallback(
-        () => Navigation.goBack(isQuickSettingsFlow ? backTo : ROUTES.WORKSPACE_TAGS_SETTINGS.getRoute(route?.params?.policyID)),
-        [isQuickSettingsFlow, backTo, route.params.policyID],
-    );
+    const goBackToTagsSettings = useCallback(() => {
+        if (isQuickSettingsFlow) {
+            Navigation.goBack(backTo);
+            return;
+        }
+        Navigation.goBack(
+            route.params.orderWeight
```

Comment:
Using this param is wrong as it is url param which will be in string form but ``0`` will be false. This caused https://github.com/Expensify/App/issues/72407